tests/test_database.py

```diff
@@ -0,0 +1,39 @@
+
+import unittest
+from app.dojo import *
+
+
+class TestDatabase(unittest.TestCase):
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_successfully_saves_state(self):
+        """ Test saves application state to Database. """
+        self.dojo.create_room('living', ["kigali"])
+        self.dojo.create_room('office', ["Django"])
+        self.dojo.add_person("Maina", "oscar", "fellow")
+        self.dojo.add_person("Ken", "musomi", "staff")
+        output = self.dojo.save_state("somedb.db")
+
+        #connect to db and create session
+        engine = create_engine('sqlite:///database/somedb.db')
```

Comment:
unittests should test functionality of the app itself, this `create_engine` section from lines 18-25 should not appear in your code. check the data exists by loading from the db and checking the loaded data is equivalent to what you expect.